apps/ui/src/networks/evm/index.ts

```diff
@@ -88,6 +88,13 @@ export const METADATA: Record<string, Metadata> = {
     apiUrl: 'https://sx-indexer.razorscan.io/subgraphs/name/snapshot-labs/sx-subgraph',
     avatar: 'https://ipfs.io/ipfs/QmRXoxMnz9ViWzQ7Ewg7dA8XtQoxp8gQ4cWjqj1Vk2ue2p/razor.png',
     blockTime: 10.7 // * Calculated from the average block time of the last 1000, 10000 blocks
+  },
+  'europa-mainnet': {
+    name: 'Europa Mainnet',
+    chainId: 2046399126,
+    apiUrl: 'http://localhost:8000/subgraphs/name/snapshot-labs/sx-subgraph',
```

Comment:
can update this as well

Reply:
updated in [519e865](https://github.com/razor-network/sx-monorepo/pull/13/commits/519e865f9396ea8204585c5fdef1794b166d4d8a)